Makefile

```diff
@@ -176,7 +176,7 @@ DEFSHAREDFS_QEMU_VIRTIOFS := virtio-fs
 DEFVIRTIOFSDAEMON := $(VIRTIOFSDBINDIR)/virtiofsd
 # Default DAX mapping cache size in MiB
 DEFVIRTIOFSCACHESIZE := 1024
-DEFVIRTIOFSCACHE ?= always
+DEFVIRTIOFSCACHE ?= none
```

Comment:
Does this have to be "auto" or "none"? Your commit message mentions "auto"

Reply:
good catch, my amend just changed the commit but missed to add the change.

Reply:
Thanks how does cache=auto work @jcvenegas ?

Reply:
@amshinde  from config docs:
#  - auto
#    Metadata and pathname lookup cache expires after a configured amount of
#    time (default is 1 second). Data is cached while the file is open (close
#    to open consistency).